spec/controllers/api/v2/searches_controller_spec.rb

```diff
@@ -305,6 +305,42 @@
                                                                  sort_by: 'date' })
       end
 
+      context 'when an audience filter is present' do
+        let(:params_with_audience) { search_params.merge(audience: 'everyone') }
+
+        it 'passes the audience filter to its ApiI4ySearch object' do
```

Comment:
Tiny thing, but the new descriptions all say "i4y". I mix up the `1` and the `i` all the time. 

```suggestion
        it 'passes the audience filter to its ApiI14ySearch object' do
```

Reply:
Oh, fun, this was an existing typo that I copy/pasted 5 additional times.  It's fixed all places now.